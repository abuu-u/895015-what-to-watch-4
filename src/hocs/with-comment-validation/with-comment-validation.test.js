import React from "react";
import renderer from "react-test-renderer";
import withCommentValidation from "./with-comment-validation.js";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withCommentValidation(MockComponent);

it(`withSubmit is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      onSubmit={() => {}}
    />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
