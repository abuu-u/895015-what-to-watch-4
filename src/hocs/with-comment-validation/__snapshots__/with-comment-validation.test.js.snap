// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSubmit is rendered correctly 1`] = `<div />`;
